### PR TITLE
Tensorboard documentation

### LINKED ISSUE #519: Documentation: Tensorboard visualization
Documentation on the usage of Tensorboard and metric visualization could be improved.

### PR DESCRIPTION
**Pull request by mansishr**
Might fix #519 
I see a scope of improvement for this documentation. Suggestions will be helpful.

### CODE REVIEW

File: docs/log_metric_callback.rst [hunk 1]
```diff
@@ -6,14 +6,15 @@
 ***********************
 Metric Logging Callback
 ***********************
-
-With the director-based workflow, you can use custom metric logging function for each task via Python\*\  API or command line interface. This function calls on the aggregator node.
-
+You can use custom metric logging function for each task via Python\*\  API or command line interface. This function calls on the aggregator node.
```
Comment (igor-davidyuk) [inline docs/log_metric_callback.rst:17]: Can we come up with a code snippet for the Metric Logging Callback? A small example could help.

Comment (psfoley) [reply docs/log_metric_callback.rst:17]: +1 . An example would really help here

**Comment by psfoley** [comment]
@mansishr Thanks for starting on this. A couple other things that should be part of this PR:
- Tensorboard logs can also be captured with the TaskRunner API, and the documentation should indicate that. A description of `write_logs` parameters in the plan (see [here](https://github.com/intel/openfl/blob/develop/openfl-workspace/workspace/plan/defaults/aggregator.yaml#L4)). When it's set to true, tensorboard logs will be written.
- Explain what metrics get saved by the aggregator (aggregator and collaborator metrics), and how to launch tensorboard from the workspace

File: docs/log_metric_callback.rst [hunk 2]
```diff
@@ -30,9 +38,53 @@ Define the function with the follow signature:
...
+  aggregator :
+    template : openfl.component.Aggregator
+    settings :
+        write_logs : true
+
+You can also write your custom callback function instead:
+
 1. Define the callback function, like how you defined in Python API, in the **src** directory in your workspace.
 
 2. Provide a way to your function with the ``log_metric_callback`` key in the ``aggregator`` section of the **plan.yaml** file in your workspace.
```
Comment (igor-davidyuk) [inline docs/log_metric_callback.rst:90]: If we have a template that implements this callback it should be mentioned here

Comment (mansishr) [reply docs/log_metric_callback.rst:90]: Moved that description here from the end of the file.

**Pull request merged**